app/controllers/FactCheckPageReviewController.scala

```diff
@@ -48,29 +48,28 @@ class FactCheckPageReviewController @Inject() (
 
   val logger: Logger = Logger(getClass)
 
-  def onPageLoad(processId: String, page: String): Action[AnyContent] = factCheckerReviewerIdentifierAction.async { implicit request =>
-    reviewService.factCheckPageInfo(processId, s"/$page") map {
-      case Right(data) =>
-        val form: Form[FactCheckPageReview] = data.result.fold(formProvider()) { answer =>
+  def onPageLoad(processId: String, pageUrl: String): Action[AnyContent] = factCheckerReviewerIdentifierAction.async { implicit request =>
```

Comment:
I wondered about this for a while, but looking at the page stanza it really is the url we use. Very Good.